cdap-app-fabric/src/main/java/io/cdap/cdap/metadata/FieldLineageAdmin.java

```diff
@@ -384,4 +384,15 @@ private Set<Field> filter(String prefix, Set<Field> fields) {
                                     field.getName().toLowerCase().startsWith(prefix.toLowerCase()))
       .collect(Collectors.toSet());
   }
+
+  /**
+   * Retrieves list of all endpoints in a given run of a pipeline.
```

Comment:
The platform layer should not know anything about pipelines.

run of a pipeline -> program run.